future.tex

```diff
@@ -108,6 +33,38 @@ \subsection{Summary}
 	}
 \end{table}
 
+The next table lists different nations in the EU and their nuclear program plans.
+The attitude is divided into five, from very negative to very positive, where 
+very negative means a complete shutdown of all nuclear facilities and 
+very positive means a rigorous expansion of nuclear power in the country.
+
+\begin{table}[h]
+	\centering
+	\caption {Future Nuclear Programs of EU Nations \cite{world_nuclear_association_nuclear_2017}}
+	\scalebox{0.70}{
+		\begin{tabular}{|c|c|c|}
+			\hline
+			Nation & Attitude & Specific Plan \\
+			\hline
+			France & neutral &Shutdown nuclear plants if they reach end of lifetime. No new construction.\\
```

Comment:
The word "shutdown" is a noun (e.g. "The national plan includes a complete shutdown."). The words "shut down", however, are a verb adverb pair (e.g. "The nation will completely shut down nuclear plants.") This particular instance should be `Shut down nuclear plants at the end of their lifetime. No new construction.`